### ISSUE TITLE
uuid column throws when defined as search field

### ISSUE THREAD

**Issue by mudlee** [ppetzold/nestjs-paginate#366]
Hi.

I have an issue when I add a PostgreSQL `uuid` or `jsonb` type field to the `searchableColumns`.
I get such errors:
```bash
QueryFailedError: operator does not exist: jsonb ~~* unknown
    at PostgresQueryRunner.query (PostgresQueryRunner.ts:299:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at SelectQueryBuilder.loadRawResults (SelectQueryBuilder.ts:3601:25)
    at SelectQueryBuilder.getRawMany (SelectQueryBuilder.ts:1573:29)
    at SelectQueryBuilder.executeEntitiesAndRawResults (SelectQueryBuilder.ts:3295:26)
    at SelectQueryBuilder.getManyAndCount (SelectQueryBuilder.ts:1776:36)
    at paginate (node_modules/nestjs-paginate/src/paginate.ts:316:28)
    at AuditLogService.get (audit-log.service.ts:600:22)
    at node_modules/@nestjs/core/router/router-execution-context.js:46:28
    at node_modules/@nestjs/core/router/router-proxy.js:9:17
```

I tried to create a querybuilder, pass it to paginate (which by default works) and tried to remove my `jsonb` field and add it manually like this:
```typescript
let queryBuilder = this.auditLogRepo.createQueryBuilder();
if (query.search) {
    queryBuilder = queryBuilder.orWhere('payload::text ILIKE :search', {
        search: `%${query.search}%`,
    });
}
``` 

This however does not work. It only works if I `delete query.search`, but obviously then it will not search for the other `searchableColumns`.

Is there any solution to this?

**Comment by mudlee** [comment]
Oh I see. Is that the same as [https://github.com/ppetzold/nestjs-paginate/issues/359](https://github.com/ppetzold/nestjs-paginate/issues/359) ?

**Comment by ppetzold** [comment]
Similar issue. JSONB columns have no coverage. Hence have not been considered with any of the features.

Happy to accept a bug fix here :)

**Comment by ppetzold** [comment]
bug fix for the uuid issue. for JSONB support, I will create a separate issue

**Comment by ppetzold** [comment]
Duplicate #436

**Comment by ppetzold** [comment]
Fixed with #442
